### ISSUE TITLE
On private members and overrides

### ISSUE THREAD

**Issue by dcodeIO** [englercj/tsd-jsdoc#9]
I have a config here that reads

```json
{
    "tags": {
        "allowUnknownTags": false
    },
    "source": {
        "include": [
            "./src/"
        ],
        "exclude": [],
        "includePattern": ".+\\.js(doc)?$",
        "excludePattern": "(^|\\/|\\\\)_"
    },
    "opts": {
        "encoding"      : "utf8",
        "recurse"       : true,
        "private"       : false,
        "lenient"       : true,
        "destination"   : "./types",
        "template"      : "./node_modules/tsd-jsdoc"
    }
}
```

When I run this with tsd-jsdoc, it generates definitions for private members with a leading `_`, which will, on a production build, be renamed to something random. Is there another way to exclude them?

Also, when I `/** @override */` a method with a different signature, the overridden signature seems not to be included in the output. Let's say I have a `Reader` that works with array-like objects and a `BufferReader` that partly requires a proper buffer, then this would be valuable information for a type checker.

**Comment by englercj** [comment]
Thanks for the report, after I finish my proto -> ts stuff I'll hope back onto my write here and ensure this gets fixed.

**Comment by englercj** [comment]
Got this fixed in v2, unfortunately there are some jsdoc bugs that make this not work 100%:

- Private constructors will still show up when `private: false` in your config
 * https://github.com/jsdoc3/jsdoc/issues/952
- Overridden methods with different parameters do not resolve properly.
 * https://github.com/jsdoc3/jsdoc/issues/1215

**Comment by englercj** [comment]
Just released `v2.0.0-beta.1`, there are still bugs (the issues I opened) but please feel free to try it out and file more bugs!
